src/ol/webgl/Helper.js

```diff
@@ -936,7 +937,7 @@ class WebGLHelper extends Disposable {
       2 / (resolution * size[0]),
       2 / (resolution * size[1]),
       -rotation,
-      -center[0],
+      -center[0] + offSetX,
```

Comment:
I think we can just add the offset by modifying `currentTransform` in the batch renderer instead of adding this to the helper API?